.github/workflows/build.yml

```diff
@@ -87,9 +87,10 @@ jobs:
           # https://github.com/ipfs/helia/pull/200
           # https://github.com/ipfs/helia-unixfs/pull/68
           # https://github.com/ipfs/helia-ipns/pull/72
-          export TAG="$(git describe --tags --abbrev=0)"
-          echo "Running tests against: $TAG"
-          git checkout "$TAG"
+          git fetch --tags
```

Comment:
There is a `fetch-depth: 0`
```suggestion
```

Reply:
- https://github.com/actions/checkout/issues/701#issuecomment-1089864361
- https://github.com/actions/checkout/issues/701#issuecomment-1133937950

Reply:
Nice! Didn't know about this. Thanks. I'm going to close this then :)